contracts/Core/interface/IParameters.sol

```diff
@@ -29,5 +29,6 @@ interface IParameters {
     function getStakeModifierHash() external view returns(bytes32);
     function getStakerActivityUpdaterHash() external view returns(bytes32);
     function getRewardModifierHash() external view returns(bytes32);
+    function getAssetCreatorHash() external view returns (bytes32);
```

Comment:
As we have update now, does AssetModifier make more sense. AssetCreator as a role is also fine, its your call

Reply:
It does both so i was kind of confused what to name the role. Maybe AssetCreatorModifier as the address with this role is both creating and modifying assets?

Reply:
Hmm dont know :joy:, Lets have AssetCreator only then.

Reply:
Alright